pkg/kwok/cmd/root.go

```diff
@@ -79,6 +80,10 @@ func NewCommand() *cobra.Command {
 		Version:       consts.Version,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			if kubeconfig != "" {
+				expanded, err := homedir.Expand(kubeconfig)
```

Comment:
Use `os.UserHomeDir()` of the standard library to get the home dir, then replace the prefix ~

Reply:
use `os.UserHomeDir()` to replace `~` can definitely fix this simple case, but why not use light-weight lib `homedir` to handle it better?

Reply:
The main point is that there is no particularly strong reason to look for the user's home directory, and if it is not available, just respond to the error and let the user decide for themselves.

Reply:
it is not reasonable to force users to use the args without the `=`, most of the docs, yaml configurations use the `=` by default.

we are already looking for users' home directory if we use `os.UserHomeDir()`, so doing it better by using the `homedir` seems more reasonable to me.

but of cause, we can use the simple `os.UserHomeDir` replacement if you insist.

Reply:
Cool, I want to wrap the `os.UserHomeDir()` into a separate utils function that wraps each place that uses the local path.